src/net.cpp

```diff
@@ -1104,82 +1104,87 @@ static void AcceptConnection(const ListenSocket &hListenSocket)
 
 char recvMsgBuf[MAX_RECV_CHUNK]; // Messages are first pulled into this buffer
 
-void ThreadSocketHandler()
+void CleanupDisconnectedNodes()
 {
```

Comment:
looks like you removed CleanupDisconnectedNodes from ThreadSocketHandler() entirely...I think you forgot to call it at the beginning as it needs to be each time do another iteration.

Reply:
dah, nvm i see it now.